style.css

```diff
@@ -0,0 +1,54 @@
+/* @media only screen and (max-width: 600px) {
+    .main-nav {
+        font-size: 0.5em;
+        padding: 0;
+    }
+} */
+body{
+  margin: 0 auto;
+}
+
+.main-nav{
+  padding: 10px 10px 10px 10px;
+  background-color: aqua;
+  width: 100%;
+  text-align: right;
+}
+
+a{
+  font-size: 2em;
+  padding: 15px;
+  text-decoration: none;
+}
+
+@media (max-width: 780px){
+  img {
+      width: 300px;
+      height: 300px;
+  }
+}
+#navbar{
+  position:fixed;
+  top:0%;
+}
+#welcome-section{
+  height:100vh;
+  background-color: #001;
+  /* background: url(https://thumbs.dreamstime.com/b/projects-concept-black-chalkboard-d-rendering-handwritten-top-view-office-desk-lot-business-office-supplies-79906734.jpg) no-repeat center fixed; */
+}
```

Comment:
I think it would be better to remove unused old code comments like this one to make your code look cleaner 👍

Reply:
- [ ] Polite,  with reason for change request 👏🏾